.gitignore

```diff
@@ -15,3 +15,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```

Comment:
I do have `.DS_Store`, etc. in my `.gitignore` but it decided to unignore. Will try again...